### PR TITLE
Meeting notes for 2017-02-16

### PR DESCRIPTION
**Pull request by harding**
Two commits:

1. I sorted the names in `_references.md` so that I could look them up more easily.  I also removed duplicate entries for Pavel (but left in duplicate entries for Jorge because they included his last name spelled with and without the accent mark).

2. Meeting notes for 2017-02-16 (please ignore the branch name, which has the wrong year).

Preview:

screenshot-127 0 0 1 4000 2017-02-22 10-21-28

### CODE REVIEW

File: _posts/en/meetings/2017-02-16-meeting.md [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+
+## 0.14.0RC1 release
+
+### Background
+
+Bitcoin Core 0.14.0 is [scheduled][#8719] to be released around March 2017. Open pull request aimed for 0.14 are [tagged with a 0.14 tag](https://github.com/bitcoin/bitcoin/pulls?q=is%3Aopen+is%3Apr+milestone%3A0.14.0).
+
+### Meeting comments
+
+General sentiment was that release readiness has improved significantly
```
Comment (btcdrak) [inline _posts/en/meetings/2017-02-16-meeting.md]: please do not break lines manually, if the pages get translated, this is an obstacle.

Comment (harding) [inline _posts/en/meetings/2017-02-16-meeting.md]: Sorry, I forgot about that rule here.  Thanks for the reminder.  I pushed a commit with line breaks removed.

File: _posts/en/meetings/2017-02-16-meeting.md [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+
+<wumpus> we need a random number of levels of randomness
+{% endhighlight %}
+
+## Participants
+ 
+| IRC nick        | Name/Nym                  |
+|-----------------|---------------------------|
+| achow101        | [Andrew Chow][]           |
+| cfields         | [cory Fields][]           |
```
Comment (achow101) [inline _posts/en/meetings/2017-02-16-meeting.md]: Capital 'C' here for Cory.

File: Gemfile.lock [hunk 3]
```diff
@@ -137,6 +137,3 @@ PLATFORMS
 DEPENDENCIES
   github-pages
   html-proofer
-
-BUNDLED WITH
-   1.11.2
```
Comment (btcdrak) [inline Gemfile.lock]: What's this for out of interest?

Comment (harding) [inline Gemfile.lock]: Whoops.  I have a different Ruby version than specified in the Gemfile and so it occasionally makes unrelated changes.  I ran `git commit -a` and pushed without checking the diff, which was bad of me.  I've forced-pushed a new commit that doesn't have this change.

Thanks for catching it!

**Comment by btcdrak** [comment]
Thanks @harding!
